lib/json.dl

```diff
@@ -40,6 +40,6 @@ typedef JsonNum = // Integer number.  The value must be in the range between the
                   // inclusive.  Values outside of this range will be truncated
                   // during serialization.
                   JsonInt{i: s128}
-                | // Always finite.
-                  // TODO.
-                  JsonFloat
+                | // NaN and infinity are not valid JSON values.
```

Comment:
I don't like this.
Perhaps JSON null would be a better choice.

Reply:
ok, null it is.